File: java/vetores/SomaVetores.java

import entities.ExercicioSomaVetores;

import java.util.Scanner;

public class SomaVetores {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int i, N, valor;

        System.out.print("Quantos valores vai ter cada vetor? ");
        N = sc.nextInt();

        ExercicioSomaVetores[] A = new ExercicioSomaVetores[N];
        ExercicioSomaVetores[] B = new ExercicioSomaVetores[N];
        int[] C = new int[N];

        System.out.println("Digite os valores do vetor A: ");
        for (i = 0; i < N; i++) {
           System.out.print("Digite um numero: ");
           valor = sc.nextInt();
           A[i] = new ExercicioSomaVetores(valor);
        }

        System.out.println("Digite os valores do vetor B: ");
        for (i = 0; i < N; i++) {
           System.out.print("Digite um numero: ");
           valor = sc.nextInt();
           B[i] = new ExercicioSomaVetores(valor);
        }

        for (i = 0; i < N; i++) {
           C[i] = A[i].getNum() + B[i].getNum();
        }

        System.out.println("VETOR RESULTANTE: ");
        for (i = 0; i < N; i++) {
           System.out.println(C[i]);
        }

        sc.close();
    }
}

File: js/vetores/somaVetores.js
const readline = require('readline-sync');

const X = Number(readline.question("Quantos valores vai ter cada vetor? "));
const A = [];
const B = [];
const C = [];

console.log("Digite os valores do vetor A:");
for(i = 0; i < X; i++) {
    const num = Number(readline.question());
    A.push(num);
}

console.log("Digite os valores do vetor B:");
for(i = 0; i < X; i++) {
    const num = Number(readline.question());
    B.push(num);
}

for(i = 0; i < X; i++) {
    C[i] = A[i] + B[i];
}

console.log("VETOR RESULTANTE:");
for(i = 0; i < X; i++) {
    console.log(C[i]);
}